tests/src/integration_test.rs

```diff
@@ -151,7 +151,8 @@ mod test {
             .context("connecting to redis")?;
         let mut con = client.get_multiplexed_async_connection().await?;
 
-        con.publish("testchannel", "some-payload").await?;
+        con.publish::<_, _, ()>("testchannel", "some-payload")
```

Comment:
Is this a change required by the toolchain update?

Reply:
yes unfortunately